python_dependencies/processing/requirements.txt

```diff
@@ -1,23 +1,17 @@
-alembic>=1.9, <2
 anndata==0.8.0
 awscli
 boto3>=1.11.17
 botocore>=1.14.17
 cellxgene-schema
 dataclasses-json
 ddtrace==2.1.4
-furl
-jsonschema
-matplotlib==3.7.3
```

Comment:
The only place matplotlib is used in the code is in **wmg_performance_profiling.py**